docs/en/drivers/ios-xcuitest-real-devices.md

```diff
@@ -245,8 +245,9 @@ for this).
     1. Settings -> Safari -> Advanced -> **Web Inspector** and **Remote Automation**
         1. Please read [Automating mobile web apps](/writing-running-appium/web/mobile-web) for more details about WebView
 1. Consider generating a provisioning profile with `.xctrunner` identifier if you do not want to generate a wildcard one for manual configuration. The `.xctrunner` config support has been added since Xcode 11. [A reference](https://github.com/appium/appium/issues/13610)
-1. Make sure the device under test is not a jaibroken one
-    - `com.apple.mobile.installation_proxy` service to manage Apps on the device [does not work](https://github.com/appium/appium-desktop/issues/1447)
+1. Make sure the device under test is not a jailbroken one
+    - Possibly it causes an issue such as [`com.apple.mobile.installation_proxy` service error to manage Apps on the device](https://github.com/appium/appium-desktop/issues/1447)
+    - Appium team does not test xcuitest driver against jailbroken devives.
```

Comment:
I'd say it makes sense to change the top sentence though

Appium team does not test xcuitest driver against jailbroken devices, so we cannot guarantee it is going to work as expected there. Use it at your own risk.

Reply:
good idea

Reply:
Changed c81e233